src/workspace-data/data-table/entity-service/EntitiesContent.js

```diff
@@ -485,7 +486,7 @@ const EntitiesContent = ({
           dataProvider,
           persist: true,
           refreshKey,
-          editable: !snapshotName && WorkspaceUtils.canEditWorkspace(workspace).value,
```

Comment:
The `!snapshotName` here isn't necessary as it's covered by the change in SnapshotContent.js to pass `editable: false`; in all the other places this component is rendered, editability is independent from `snapshotName` and can be set on its own.